keyboards/polykybd/tools/status_oled_preview.py

```
@@ -137,15 +140,61 @@ def draw_glyph_half(setpix, font, x, y, cp):
+        x += g['xa']
+    return x0, x1, y0, y1
+
+
+def build_fw_confirm_panel(side, small):
+    """FW-2 unsigned-image confirmation screen — mirror of oled_helper.c's
+    oled_fw_confirm_screen(). Three lines, each centred in its own H/3 band from
+    its own bbox, so a line with a descender sits level rather than being pushed
+    by the tallest line in the set. `side` is 'L' or 'R' — each half names its
+    OWN key, exactly as the firmware does."""
+    pts = []
+    setp = lambda px, py: pts.append((px, py))
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `setp` as a `def`

(E731)